tensorflow/core/lib/random/random_distributions.h

```diff
@@ -56,13 +98,15 @@ PHILOX_DEVICE_INLINE Int SignedAdd(Int a,
 //              actual returned sample type.
 //   RealType: the data type of the real numbers that will be returned by the
 //             distribution. This could be either float or double for now.
+//   IsVec: mark this UniformDistribution can be vectorized or not by SIMD on
```

Comment:
Can we rename this argument to `Vectorized`?

Reply:
Done.